src/components/volunteer/VolunteerFormCalendar.tsx

```diff
@@ -399,6 +408,52 @@ export const VolunteerFormCalendar = ({
   );
 };
 
+const fileFilters = (
+  fileRegex: any,
+  calendarData: {
+    id: number;
+    content: { files: any[] };
+  }
+) =>
+  JSON.stringify(
+    calendarData?.content?.files
+      .map(({ file_name: infoText, mime_type: mimeType, url: uri, id: fileId }: any) => {
+        const isFile = fileRegex.test(infoText);
+
+        if (isFile) return { entryId: calendarData?.id, fileId, infoText, mimeType, uri };
+        else return;
+      })
+      .filter((otherFiles: any) => otherFiles != null)
```

Comment:
when filtering for `null` here there need to be `null` returned in the else case of the `map` before or am i wrong?

Reply:
Yeah, you're right, I missed it. 🤦‍♂️